codemirror-wp.php

```diff
@@ -25,6 +25,18 @@ public static function register_scripts() {
 		wp_register_script( 'codemirror-addon-hint-sql',        plugins_url( "CodeMirror/addon/hint/sql-hint.js", __FILE__ ),        array( 'codemirror-addon-hint-show', 'codemirror-mode-sql' ), SELF::CODEMIRROR_VERSION );
 		wp_register_script( 'codemirror-addon-hint-xml',        plugins_url( "CodeMirror/addon/hint/xml-hint.js", __FILE__ ),        array( 'codemirror-addon-hint-show', 'codemirror-mode-xml' ), SELF::CODEMIRROR_VERSION );
 
+		// The linting engines for the lint addons...
+		wp_register_script( 'csslint',  'http://csslint.net/js/csslint.js' );
+		wp_register_script( 'htmlhint', 'http://htmlhint.com/js/htmlhint.js' );
+		wp_register_script( 'jshint',   '//ajax.aspnetcdn.com/ajax/jshint/r07/jshint.js' );
+		wp_register_script( 'jsonlint', 'https://rawgithub.com/zaach/jsonlint/79b553fb65c192add9066da64043458981b3972b/lib/jsonlint.js' );
```

Comment:
Agreed.  I'm planning on setting them as externals before actually merging this.  Or maybe just importing wholesale.  Unsure.